var/spack/repos/builtin/packages/mmv/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+
+class Mmv(MakefilePackage):
+    """Move/Copy/Append/Link multiple files mmv is a program to
+    move/copy/append/link multiple files according to a set of wildcard
+    patterns. This multiple action is performed safely, i.e. without any
+    unexpected deletion of files due to collisions of target names with
+    existing filenames or with other target names. """
+
+    homepage = "https://packages.debian.org/source/buster/mmv"
+    url      = "http://deb.debian.org/debian/pool/main/m/mmv/mmv_1.01b.orig.tar.gz"
+
+    version('1.01b', sha256='0399c027ea1e51fd607266c1e33573866d4db89f64a74be8b4a1d2d1ff1fdeef')
+
+    patch('better-diagnostics-for-directories-584850.diff')
+    patch('format-security.diff')
+    patch('man-page-examples.diff')
+    patch('man-page-fixes.diff')
+    patch('man-page-warning-149873.diff')
+    patch('patches-as-of-mmv-1.01b-15.diff')
+    patch('utime.diff')
```

Comment:
Some of these patches refer to a debian bug number (so are probably easy to find but a link would still be useful). Some have a commit message (but no identifier or link). In the case of `utime.diff` there is neither. In each of these cases is it possible to link to the original issue, so that users could figure out when the patches are no longer needed?

Reply:
I can add the links to Debian bug numbers. `utime.diff` was added by me since an include header was missing.

`mmv` is very useful, and exists in each major Linux distribution. However, it does not have an independent maintainer, and the software was released at a time (1989) where software repositories did not yet exist; instead, `mmv` was posted to Usenet. Hence there is no real "upstream", and thus I'm pointing only to Debian.

Debian distributes it as "original tarball" plus patches, and I mimic this here. I don't expect these patches to go away. An alternative would be to obtain two tarballs, the original code as well as the set of Debian patches, and construct the source code from there. Or one could put `mmv` into a new Github repo and point to it (the `mmv` license is GPL). Do you have a preference?

Reply:
That makes sense. In that case I think this package can be merged as is (the links for the bug numbers are unnecessary).